stylesheets/commons/FilterButtons.less

```diff
@@ -58,6 +58,52 @@ Author(s): Elysienna
 	animation-duration: 0.8s;
 }
 
+.dota2-test {
+	.filter-buttons {
+		display: flex;
+		justify-content: start;
+		flex-flow: row wrap;
+		font-size: 10pt;
```

Comment:
Is this partly old css? If so then I might be a good time to clean this up. For example pt is not a desired unit for web, so I'd convert it to rem. I would also change margin and padding to rem units.

Reply:
Updated